docs/_docs/user-guide/eldritch.md

```diff
@@ -823,6 +823,14 @@ sys.shell("ls /nofile")
 
 ### time.sleep
 
+`time.format_to_epoch(input: str, format: str) -> int`
```

Comment:
Can you provide examples of the formats as well as a link to the docs on all acceptable formats whether that's the crate specific or the generic time formats.